sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -153,6 +156,56 @@ public partial class SharedSearchServicePrivateLinkResourceData : Azure.Resource
+    }
+    public partial class EncryptionWithCmk
+    {
+        public EncryptionWithCmk() { }
+        public Azure.ResourceManager.Search.Models.SearchEncryptionComplianceStatus? EncryptionComplianceStatus { get { throw null; } }
+        public Azure.ResourceManager.Search.Models.SearchEncryptionWithCmk? Enforcement { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PrivateLinkServiceConnectionProvisioningState : System.IEquatable<Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public PrivateLinkServiceConnectionProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Deleting { get { throw null; } }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Incomplete { get { throw null; } }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Succeeded { get { throw null; } }
+        public static Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState Updating { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState left, Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState left, Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public enum SearchEncryptionComplianceStatus
+    {
+        Compliant = 0,
+        NonCompliant = 1,
+    }
+    public enum SearchEncryptionWithCmk
```

Comment:
We might need to change this enum type to `SearchEncryptionWithCmkEnforcement` from the property it is being used.